### PR TITLE
Add test environment and a bunch of tests

### PR DESCRIPTION
**Pull request by busykoala**
To be able to test the pdf converter I add a test setup using pytest in this Pull.

Also I add a couple of tests aiming to test the basic functionality of the application.
More testing will be required in the future though.

### CODE REVIEW

File: pdf_converter/test/test_md_to_pdf.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+def test_converting_md_to_pdf(tmpdir):
+    package_path = pdf_converter.__path__[0]
+    input_file_dir = package_path + '/test/assets/testfile.md'
+    output_path = str(tmpdir) + '/testfile.pdf'
+
+    file_instance = PdfConverter(input_file_dir)
+    file_instance.set_file_format()
+    file_instance.convert_markdown(output_path)
+
+    # Not that we converted the file we expect
```
Comment (mbaechtold) [inline pdf_converter/test/test_md_to_pdf.py]: The sentence is kind of confusing...

Comment (busykoala) [reply pdf_converter/test/test_md_to_pdf.py]: I changed the sentence a little to make it less confusing.

File: pdf_converter/test/test_md_to_pdf.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    package_path = pdf_converter.__path__[0]
+    input_file_dir = package_path + '/test/assets/testfile.md'
+    output_path = str(tmpdir) + '/testfile.pdf'
+
+    file_instance = PdfConverter(input_file_dir)
+    file_instance.set_file_format()
+    file_instance.convert_markdown(output_path)
+
+    # We expect that there is a converted file
+    # now. It should have the same dir as the input.
```
Comment (mbaechtold) [inline pdf_converter/test/test_md_to_pdf.py]: If it should have the same dir as the input, shouldn't we then make an assertion which makes sure it really does?

Comment (busykoala) [reply pdf_converter/test/test_md_to_pdf.py]: Oh, you are right. Actually the sentence is wrong again. Of course it is not the same directory, but the one given as a user input. I'll correct that.

Also I have to find out why the tests are failing now. I didn't change anything on the code...

Comment (busykoala) [reply pdf_converter/test/test_md_to_pdf.py]: Tests were failing of a broken (new) version of buggy. They had another release so it should start working again.

**Comment by busykoala** [comment]
I added an issue to add more and better tests including checking the pdf file itself.
https://github.com/busykoala/pdf_converter/issues/16

**Pull request merged**